docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectIidChangePoint.cs

```diff
@@ -46,35 +48,81 @@ public static void Example()
             // Convert data to IDataView.
             var dataView = ml.Data.LoadFromEnumerable(data);
 
-            // Setup estimator arguments
+            // Setup IidSpikeDetector arguments
             string outputColumnName = nameof(ChangePointPrediction.Prediction);
             string inputColumnName = nameof(IidChangePointData.Value);
 
-            // The transformed data.
-            var transformedData = ml.Transforms.DetectIidChangePoint(outputColumnName, inputColumnName, 95, Size / 4).Fit(dataView).Transform(dataView);
+            // Time Series model.
+            ITransformer model = ml.Transforms.DetectIidChangePoint(outputColumnName, inputColumnName, 95, Size / 4).Fit(dataView);
 
-            // Getting the data of the newly created column as an IEnumerable of ChangePointPrediction.
-            var predictionColumn = ml.Data.CreateEnumerable<ChangePointPrediction>(transformedData, reuseRowObject: false);
+            // Create a time series prediction engine from the model.
+            var engine = model.CreateTimeSeriesPredictionFunction<IidChangePointData, ChangePointPrediction>(ml);
```

Comment:
>CreateTimeSeriesPredictionFunction [](start = 31, length = 34)

I believe the more complete examples should go in the other repo, this project is just for API reference.

Reply:
I don’t see how this is difference from any other API example. These are the only examples we have for stateful prediction engine and should exist in Samples.

Reply:
spoke offline with @sfilipi and we agreed this can go here and have it linked to checkpoint API.